groclient/cli.py

```diff
@@ -67,14 +106,18 @@ def main():  # pragma: no cover
         and not args.region
         and not args.partner_region
     ):
-        ds = client.pick_random_data_series(client.pick_random_entities())
```

Comment:
Actually I see you removed them from the class, there must be a reason..

Reply:
Yup I removed them from the class, because they're specific to the `gro_client` command line tool. I don't think there is much value in the class having a randomize method. It's like what's stated [in the README](https://github.com/gro-intelligence/api-client#examples):

> The gro_client command line interface does a keyword search for the inputs and finds a random matching data series. It displays the data series it picked in the command line and writes the data points out to a file in the current directory called gro_client_output.csv. This tool is useful for simple queries, but anything more complex should be done using the provided Python packages.

Reply:
Ok, that makes sense